go/internal/controller/translator/agent/testdata/outputs/agent_with_proxy.json

```diff
@@ -0,0 +1,311 @@
+{
+  "agentCard": {
+    "capabilities": {
+      "pushNotifications": false,
+      "stateTransitionHistory": true,
+      "streaming": true
+    },
+    "defaultInputModes": [
+      "text"
+    ],
+    "defaultOutputModes": [
+      "text"
+    ],
+    "description": "",
+    "name": "agent_with_proxy",
+    "skills": null,
+    "url": "http://agent-with-proxy.test:8080",
+    "version": ""
+  },
+  "config": {
+    "description": "",
+    "http_tools": [
+      {
+        "params": {
+          "headers": {
```

Comment:
i don't think we want any visible changes in the agent card. ideally using a proxy should be internal to the agent